File: c++/Easy/Part001/Odd Integers In Range.cpp
#include <iostream>
using namespace std;

int main() {
    int X, Y;
    
    // Input for X and Y
    cin >> X;
    cin >> Y;

    // Loop through numbers between X and Y
    for (int i = X + 1; i < Y; i++) {
        if (i % 2 != 0) {
            cout << i << " ";
        }
    }

    return 0;
}

File: Java/Easy/Part001/Odd Integers in Range.java
import java.util.Scanner;

public class Hello {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        // Input for X and Y
        int X = scanner.nextInt();
        int Y = scanner.nextInt();
        
        // Loop through numbers between X and Y
        for (int i = X + 1; i < Y; i++) {
            if (i % 2 != 0) {
                System.out.print(i + " ");
            }
        }
        
        scanner.close();
    }
}
